examples/slack/README.md

```diff
@@ -0,0 +1,195 @@
+           # Run Infracost breakdown and save to a tempfile, namespaced by this project, PR, workspace and dir
+           - run: |
+               infracost breakdown --path=$SHOWFILE \
+                                   --format=json \
+                                   --log-level=info \
+                                   --out-file=/tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-infracost.json
+           # Use Infracost comment to create a comment containing the results for this project
+           - run: |
+               infracost comment gitlab --repo $BASE_REPO_OWNER/$BASE_REPO_NAME \
+                                        --merge-request $PULL_NUM \
+                                        --path /tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-infracost.json \
+                                        --behavior new \
+                                        --gitlab-token $GITLAB_TOKEN
+           # Use Infracost output command to create a message payload for Slack
+           - run: |
+               infracost output --path /tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-infracost.json \
+                                --format slack-message \
+                                --out-file /tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-slack-message.json
+           # Use cURL to send a Slack message via a webhook
+           - run: |
+               # Skip posting to Slack if there's no cost change
+               cost_change=$(cat /tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-infracost.json | jq -r "(.diffTotalMonthlyCost // 0) | tonumber")
+               if [ "$cost_change" = "0" ]; then
+                 echo "Not posting to Slack since cost change is zero"
+                 exit 0
+               fi
+
+               if [ -z "$SLACK_WEBHOOK_URL" ]; then
+                 echo "No \$SLACK_WEBHOOK_URL variable set."
+                 exit 1
+               fi
+
+               curl -X POST -H "Content-type: application/json" -d @/tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-slack-message.json $SLACK_WEBHOOK_URL
+   ```
+
+## Running with Azure Repos
+
+1. Update your setup to use the [infracost-atlantis](https://hub.docker.com/r/infracost/infracost-atlantis) Docker image.
+2. You'll need to pass the following custom env vars into the container:
+   ```sh
+   AZURE_ACCESS_TOKEN=<your-azure-devops-access-token-or-pat>
+   AZURE_REPO_URL=<your-azure-repo-url> # i.e., https://dev.azure.com/your-org/your-project/_git/your-repo
+   INFRACOST_API_KEY=<your-infracost-api-token>
+   SLACK_WEBHOOK_URL: <your-slack-webhook-url>
+   ```
+3. Add the following yaml spec to `repos.yaml` or `atlantis.yaml` config files:
+   ```yaml
+   repos:
+     - id: /.*/
+       workflow: terraform-infracost
+       post_workflow_hooks:
+         # Clean up any files left over from the run
+         - run: |
+             rm /tmp/${BASE_REPO_OWNER//\//-}-$BASE_REPO_NAME-$PULL_NUM-infracost.json
```

Comment:
Have to do this `${BASE_REPO_OWNER//\//-}` for Azure Repos - the owner comes in as `infracost/base` (using our demo repo) and it fails to create files.